src/current/v25.2/detect-hotspots.md

```diff
@@ -0,0 +1,161 @@
+- Check if the CPU usage of the hottest node is 20% or more above the cluster average. For example, node `n5`, represented by the green line in the following **CPU Percent** graph, hovers at around 87% at time 17:35 compared to other nodes that hover around 20% to 25%.
+  
+<img src="{{ 'images/v25.2/detect-hotspots-cpu-percent.png' | relative_url }}" alt="graph of CPU Percent utilization per node showing hot key" style="border:1px solid #eee;max-width:100%" />
+
+- Is there a node with a maximum value that is a clear outlier in the cluster for the CPU percent metric?
+
+  - If **Yes**, note the ID of the [hot node]({% link {{ page.version.version }}/understand-hotspots.md %}#hot-node) and the time range when it was hot. Proceed to check for a [`popular key detected `log](#a-popular-key-detected).
+  - If **No**, investigate other possible causes for the metrics outlier.
+
+## Step 2. Check for existence of `no split key found` log
+
+The [`no split key found` log]({% link {{ page.version.version }}/load-based-splitting.md %}#monitor-load-based-splitting) is emitted in the [`KV_DISTRIBUTION` log channel]({% link {{ page.version.version }}/logging-overview.md %}#logging-channels). This log is not associated with a specific event type, but includes an unstructured message such as:
+
+```
+I250523 21:59:25.755283 31560 13@kv/kvserver/split/decider.go:298 ⋮ [T1,Vsystem,n5,s5,r1115/3:‹/Table/106/1/{113338-899841…}›] 2979  no split key found: insufficient counters = 0, imbalance = 20, most popular key occurs in 36% of samples, access balance right-biased 98%, popular key detected, clear direction detected
+```
+
+In the preceding log example, the tag section in square brackets provides the following information:
+
+- node ID: the node ID is 5 from `n5`.
+- range ID: the range is 1115 from `r1115`.
+
+The timestamp is at the beginning of the log is `250523 21:59:25.755283`.
+
+The unstructured message ends in either of these string combinations:
+
+1. `popular key detected, clear direction detected`
+1. `popular key detected, no clear direction`
+1. `no popular key, clear direction detected`
+1. `no popular key, no clear direction`
+
+### A. `popular key detected`
+
+- To check whether a `popular key detected` log exists, search for `popular key detected` in the `KV_DISTRIBUTION` logs on the hot node you noted in Step 1 in the time range that you noted.
+
+- Once you identify a relevant log, note the range ID in the tag section of the log.
+
+{{site.data.alerts.callout_info}}
+There may be false positives of the `popular key detected` log.
+{{site.data.alerts.end}}
+
+- The outlier was in the latch conflict wait durations metric. Does a `popular key detected` log exist?
+
+  - If **Yes**, it is a [write hotspot]({% link {{ page.version.version }}/understand-hotspots.md %}#write-hotspot). Note the range ID of `popular key detected` log and proceed to find the corresponding [hot ranges log](#step-3-find-hot-ranges-log).
```

Comment:
Here I'd say `it may be a hotspot` rather than `it is a hotspot`. Same thing below (only for popular key detected log).